pages/entity/_type/_.vue

```diff
@@ -141,6 +144,18 @@
         }));
       }
 
+      let contentfulClient;
+      if (query.mode === 'preview' && process.env['CTF_CPA_ACCESS_TOKEN']) {
```

Comment:
Let's not check this here, but instead replace the whole if/else, both here and in pages/index.vue, with just:
```js
contentfulClient = createClient(query.mode);
```

If anywhere should be responsible for testing for the presence of the preview API token, it's the plugin, but we could even omit it from there.